about_binaries.exs

```diff
@@ -31,4 +31,66 @@ defmodule AboutBinaries do
     contatenated = "Hello " <> "world"
     assert contatenated == __?
   end
+
+  @tag :focus
+  think "Numbers can be represented as bits" do
+    bit_representation = fn (integer) -> Integer.to_string(integer, 2) end
+    assert bit_representation.(1) == "1"
+    assert bit_representation.(2) == "10"
+    assert bit_representation.(3) == "11"
+    assert bit_representation.(4) == "100"
+  end
+
+  @tag :focus
+  think "255 is the max value we can represent with 8 bits (1 byte)" do
+    bit_representation = fn (integer) -> Integer.to_string(integer, 2) end
+    assert bit_representation.(255) == "11111111"
+    assert String.length(bit_representation.(255)) == 8
+    assert bit_representation.(256) == "100000000"
+    assert String.length(bit_representation.(256)) == 9
+  end
+
+  @tag :focus
+  think "Unicode defines far more than 255 codepoints" do
+    assert ?λ == 955
+    assert ?™ == 8482
+  end
+
+  # These would be be simple, but misleading; actually characters whose
+  # codepoint is > 128 require more than 1 byte, because UTF8 always uses at
+  # least 1 bit of metadata per byte (a leading 0 in the case of "a", to say
+  # "this byte is not part of a sequence; it encodes the whole codepoint")
+  #
+  # @tag :focus
+  # think "Characters whose codepoint is <= 255 use 1 byte" do
+  #   assert ?a == 97
+  #   assert String.length("a") == 1
+  #   assert byte_size("a") == 1
+  #   assert "a" === <<97>>
+  # end
+  #
+  # @tag :focus
+  # think "Characters whose codepoint is > 255 require more than 1 byte" do
+  #   assert ?λ == 955
+  #   assert String.length("λ") == 1
+  #   assert byte_size("λ") == 2
+  #   assert "λ" === <<206, 187>>
+  # end
+
+  @tag :focus
+  think "Therefore, most Unicode codepoints require more than 1 byte to represent" do
+    assert ?λ == 955
+    assert String.length("λ") == 1
+    assert byte_size("λ") == 2
```

Comment:
OoO i like this a lot. The length is the number of codepoints (characters), but the byte size isn't. Nice!